pro/src/components/AddressManual/AddressManual.tsx

```diff
@@ -0,0 +1,93 @@
+import { parseDms } from 'dms-conversion'
+import { useField, useFormikContext } from 'formik'
+import { useEffect } from 'react'
+
+import { Callout } from 'components/Callout/Callout'
+import { CalloutVariant } from 'components/Callout/types'
+import { FormLayout } from 'components/FormLayout/FormLayout'
+import fullLinkIcon from 'icons/full-link.svg'
+import { ButtonLink } from 'ui-kit/Button/ButtonLink'
+import { ButtonVariant } from 'ui-kit/Button/types'
+import { TextInput } from 'ui-kit/form/TextInput/TextInput'
+import { getCoordsType } from 'utils/coords'
+
+import styles from './AddressManual.module.scss'
+
+export const AddressManual = (): JSX.Element => {
+  const [coords, coordsMeta] = useField('coords')
+
+  const { setFieldValue } = useFormikContext()
+
+  useEffect(() => {
+    if (coords.value && !coordsMeta.error) {
+      const coordsType = getCoordsType(coords.value)
+
+      if (coordsType === 'DD') {
+        const [latitude, longitude] = coords.value.split(', ')
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('latitude', latitude)
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('longitude', longitude)
+      } else if (coordsType === 'DMS') {
+        const [latitude, longitude] = coords.value.split(' ')
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('latitude', parseDms(latitude))
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('longitude', parseDms(longitude))
+      } else {
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('latitude', '')
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        setFieldValue('longitude', '')
+      }
+    }
+  }, [coords.value, coordsMeta.error, setFieldValue])
+
+  return (
+    <div className={styles['address-manual-wrapper']}>
+      <FormLayout.Row>
+        <TextInput label="Adresse postale" name="street" />
+      </FormLayout.Row>
+      <div className={styles['inline-fields']}>
+        <FormLayout.Row inline className={styles['field-cp']}>
+          <TextInput label="Code postal" name="postalCode" maxLength={5} />
+        </FormLayout.Row>
+        <FormLayout.Row inline className={styles['field-city']}>
+          <TextInput label="Ville" name="city" />
+        </FormLayout.Row>
+      </div>
```

Comment:
Tu as vraiment besoin de 2 row, et la div qui wrap ?
```suggestion
        <FormLayout.Row inline className={styles['field-cp']}>
          <TextInput label="Code postal" name="postalCode" maxLength={5} />
          <TextInput label="Ville" name="city" />
        </FormLayout.Row>
```